nomad/job_endpoint.go

```diff
@@ -1027,8 +1027,9 @@ func (j *Job) Scale(args *structs.JobScaleRequest, reply *structs.JobRegisterRes
 	if job == nil {
 		return structs.NewErrRPCCoded(404, fmt.Sprintf("job %q not found", args.JobID))
 	}
-	if job.Type == structs.JobTypeSystem {
-		return structs.NewErrRPCCoded(http.StatusBadRequest, `cannot scale jobs of type "system"`)
+
+	if job.Type == structs.JobTypeSystem && *args.Count > 1 {
```

Comment:
I know this is where the current JobTypeSystem validation was located.. But should it really be in the `args.Validate()` method?

Reply:
It would make more sense, but the args do not include the job type, so we need to look the job up first before we can validate it. We could add the job specs as an argument to the validation, but we might be making calls for args that are straight up invalid. Thinking more about it, Im not sure how expensive is that call, do you think it is a fair trade off? Also it will mean changing the signature of the validation, Im not sure how big of a change that would be.

Reply:
I think I mixed up the fields in the `JobScaleRequest` and thought job was included, but that was wrong.  Looking at it again after reading the above response makes we feel like your current way of doing this might be the better than what I suggested.